conda_rpms/tests/integration/gitenv_install.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+
+REPO_ROOT=$(cd "$(dirname ${0})/../../.."; pwd;)
+
+cat << 'EOF' | docker run -i \
+                        -v ${REPO_ROOT}:/repo \
+                        -a stdin -a stdout -a stderr \
+                        centos:6 \
+                        bash || exit ${?}
+
+
+export MINICONDA_DIR=${HOME}/miniconda
+
+# Install conda
+# -------------
+yum install -y wget
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --no-verbose
+bash Miniconda3-latest-Linux-x86_64.sh -b -p ${MINICONDA_DIR} && rm -f Miniconda*.sh
+
+export PATH=$MINICONDA_DIR/bin:$PATH
+
+conda config --set always_yes yes --set changeps1 no --set show_channel_urls yes
+
+
+# Create an environment with the noarch package `tqdm`
+# ----------------------------------------------------
+# This will ensure that the noarch package and all it's dependencies are
+# downloaded and in the cache.
+conda create -n test-env tqdm -c conda-forge
```

Comment:
@lbdreyer Did you want to be explicit about the version of Python used here? As you're relying on it being `Python3.6` in [line 55](https://github.com/SciTools-incubator/conda-rpms/pull/9/files#diff-3a6897e67b316676ab51b75eee9c9cc7R55).

So we have:
```shell
conda create -n test-env -c conda-forge tqdm python=3.6
```

Reply:
Agreed to pin the `tqdm` version to what we need and want